CONTRIBUTING.md

```diff
@@ -93,6 +93,10 @@ New features are welcome! Either as requests or proposals.
 Don't get discouraged! We estimate that the response time from the
 maintainers is around a day or so.
 
+## Tools
+
+As Stryker uses the Abstract Syntax Tree (AST) to create mutations, you recommend you to use [ASTExplorer](https://astexplorer.net/) to get familiar with the concepts involved.
```

Comment:
we recommend you? Or it is recommended. Or make two sentences.
Stryker uses the Abstract Syntax Tree (AST) to create mutations.
If you want to learn more about it, check out [ASTExplorer](https://astexplorer.net/) to get familiar with the concepts involved.

What do you think about it? :)

Reply:
See the new commit, I changed it already ;)

Reply:
Well then for me it's good. I am just thinking about 'concepts involved' or 'involved concepts', but  I guess both are correct